source/geh_common/src/geh_common/data_products/electricity_market_measurements_input/missing_measurements_log_metering_point_periods_v1.py

```diff
@@ -17,10 +17,14 @@
         # Metering point resolution: PT1H/PT15M
         T.StructField("resolution", T.StringType(), not nullable),
         #
-        # UTC time
+        # First time the metering point is connected or disconnected. UTC time
         T.StructField("period_from_date", T.TimestampType(), not nullable),
         #
-        # UTC time
+        # The date where the the metering point is closed down. UTC time
         T.StructField("period_to_date", T.TimestampType(), nullable),
     ]
 )
+"""
+Metering point periods for missing measurements log. These are the periods where the metering points are active,
+and can receive measurements. It includes all metering point types except internal_use (D99) and those calculated by Datahub (sub_type=calculated) 
```

Comment:
So, new metering point types being added will automatically be included?

Reply:
Yes. The business have a process for handling new types. One mandatory thing to consider is whether it should be included in missing measurements log. If yes, then no code changes are required. If no, then a change request is created.